api/core/v1alpha1/cnset_types.go

```diff
@@ -34,6 +34,12 @@ const (
 	CNStoreStateUp       string = "Up"
 )
 
+const (
+	ContainerPythonUdf             string = "python-udf"
+	ContainerPythonUdfDefaultPort  int    = 50051
+	ContainerPythonUdfDefaultImage string = "composer000/mo-python-udf-server:latest" // TODO change it
```

Comment:
where's the dockerfile of this image?

Reply:
Here: https://github.com/matrixorigin/matrixone/blob/main/optools/images/pythonserver/Dockerfile